src/web/mjs/connectors/Manhuaes.mjs

```diff
@@ -0,0 +1,59 @@
+import Connector from '../engine/Connector.mjs';
+import Manga from '../engine/Manga.mjs';
+
+export default class ManhuaES extends Connector {
+
+    constructor() {
+        super();
+        super.id = 'manhuaes';
+        super.label = 'Manhua ES';
+        this.tags = [ 'manga', 'English' ]; //should I add 'Webtoon' tag as well?
```

Comment:
When the website provides mangas, then set `manga`
When the website provides webtoons, then set `webtoon`
When the website provides both, then add `manga` and `webtoon`

Use lowercase for tags, otherwise there will be two groups `english` and `English` with the latter only containing this single website